### DISCUSSION TITLE
Gifs turn into a video player

### DISCUSSION THREAD

**Discussion by Pareuelle** [AprilSylph/XKit-Rewritten#661] [Help Desk] [answered]
Hi, hopefully this is the right place to post this.

A couple days ago I noticed gifs on my dashboard started turning into mini video players. If I click on it, the gif loads like it usually would.

This happens randomly. If a post has multiple gifs, sometimes all of them will load or sometimes only a few will. Sometimes a post with a gif will load normally but if I refresh the page it turns into a video player.

Disabling XKit Rewritten fixes the issue.

![50bddd02e0e80b07889833e16608abff](https://user-images.githubusercontent.com/109317984/179137581-6e0ec861-ccd7-4e72-8717-3d07e696ad84.png)

**Comment by marcustyphoon** [comment, marked as answer]
I believe this will be fixed in the release this week! (0f624931973ca7492b1723dcf07962a647395818) In the meantime, you can temporarily disable Vanilla Videos.

**Reply by Pareuelle** [reply]
I feel like an idiot for not even checking the VIDEO setting. Thank you for the quick reply! I have a hard time navigating Github so I apologize if this was obviously available knowledge.
